.github/workflows/ci.yml

```diff
@@ -115,13 +130,6 @@ jobs:
           title: "Update cassette-diff-PR${{ github.event.pull_request.number }}-${{ env.TIMESTAMP_COMMIT }}"
           body: "This PR updates the cassettes. Please merge it."
 
-
-      - name: Check PR
-        if:  ${{ env.TIMESTAMP_COMMIT != null }}
-        run: |
-          echo "Pull Request Number - ${{ steps.create_pr.outputs.pull-request-number }}"
-          echo "Pull Request URL - ${{ steps.create_pr.outputs.pull-request-url }}"
-
       - name: Comment PR URL in the current PR
         if:  ${{ env.TIMESTAMP_COMMIT != null }}
         uses: thollander/actions-comment-pull-request@v2
```

Comment:
This is the piece that weirds me out, but I understand that isn’t what this pr is about